docs/configuration/providers/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Configuration Providers
+layout: default
+---
+
+Configuration can typically come from different sources. For exemple, command line or desktop apps usually store the configuration in files the the user home directory; server deployments might use a central configuration database; or during testing, the configuration might be hardcoded in the test.
+
+To implement this behavior, Powsybl defines a Configuration Provider interface: `com.powsybl.commons.config.PlatformConfigProvider`. You can use the implementations shipped with Powsybl or implement your own.
+
+Note: The `iTools` and `GSE` distributions automatically use the `ClassicPlatformConfigProvider` so you don't have to think about it when using these programs. But when using Powsybl as a plain java library, the developper must choose a configuration provider. To choose a configuration provider, ensure that it is the only implementation of PlatformConfigProvider available to Java's ServiceLoader. Otherwise, you will get the following error:
```

Comment:
@jonenst could you adapt itools-packager tutorial to add this new dependency. Except if you did it recently, it doesn't work properly since this change.